docs/contributing/templates/c2c-source-and-app.md

```diff
@@ -0,0 +1,170 @@
+
+### Source configuration
+
+\Insert steps to configure the Source in the Sumo Logic UI\
+
+Example: https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dropbox-source/#create-a-dropboxsource
+
+#### Metadata fields
+
+\Insert meta deta fields in the Sumo Logic UI. Update the below table accordingly.\
+
+| Field | Value | Description |
+| :--- | :--- | :--- |
+| {{`field`}} | {{value}} | {{Description}} |
+
+#### JSON schema
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for more details. 
+
+| Parameter | Type | Value | Required | Description |
+|:--|:--|:--|:--|:--|
+| schemaRef | JSON Object  | `{"type":"/*c2c-name*/"}` | Yes | Define the specific schema type. |
+| sourceType | String | `"Universal"` | Yes | Type of source. |
+| config | JSON Object | [Configuration parameters](#config-object) | Yes | Source type specific values. |
+
+#### Config Object
+
+\Add information about the configuration parameters. Update the below table accordingly.\
+
+| Parameter | Type | Required | Default | Description | Example |
+|:--|:--|:--|:--|:--|:--|
+| name | String | Yes | `null` | Type a desired name of the source. The name must be unique per Collector. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_source`. | `"mySource"` |
+| description | String | No | `null` | Type a description of the source. | `"Testing source"`
+| category | String | No | `null` | Type a category of the source. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_sourceCategory`. See [best practices](/docs/send-data/best-practices) for details. | `"mySource/test"`
+| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field _siemForward to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
+| {{`Parameter`}} | {{Type}} | {{Yes/No}} | `null` | {{Description}} |  {{Example}} |
+
+#### JSON example
+
+\Create and add the JSON config in the dropbox and import it here.\
+
+<CodeBlock language="json">component-name</CodeBlock>
+
+#### Terraform example
+
+\Create and add the Terraform config in the dropbox and import it here.\
+
+<CodeBlock language="json">component-name}</CodeBlock>
+
+### Troubleshooting
+
+\This section doesn't apply to all sources; use only where needed\
+
+### FAQ
+
+:::info
+Click [here](/docs/c2c/info) for more information about Cloud to Cloud sources.
+:::
+
+## Installing the /app name/ app
+
+\Reuse the below content\
+
+{@import ../../reuse/filter-dashboards.md}
+
+## Viewing /app name/ dashboards
+
+\Reuse the below content\
+
+{@import ../../reuse/apps/app-install.md}
```

Comment:
You'll need to update the /reuse file import syntax to align with Docusaurus v3.